src/db/migrations/20250219182508_add_notifications_table.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function up(knex) {
+  return knex.schema.createTable("notifications", (table) => {
+    table.increments("notification_id").primary();
+    table.string("label").nullable();
+    table.string("title").nullable();
+    table.string("description").nullable();
+    table.string("small_image_path").nullable();
+    table.string("big_image_path").nullable();
```

Comment:
So... Where are these paths supposed to point? It sounds like they might need to be in our repository, which means they would need an engineer to add, which means... That this might actually get more of a hassle than it would be if an engineer could just add the whole notification in one go, like we did on Relay?

Reply:
I moved this discussion to chat, since it'll be out of scope for this PR.